docs/website/mmp-errors.md

```diff
@@ -62,13 +62,26 @@ The easiest way to get exact version information is to use the **Xamarin Studio*
 
 ### <a name="MM0068">MM0068: Invalid value for target framework: {0}.
 
+### <a name="MM0071">MM0071: Unknown platform: *. This usually indicates a bug in Xamarin.Mac; please file a bug report at http://bugzilla.xamarin.com with a test case.
```

Comment:
Minor: https instead of http.

Reply:
There appears to be 42 instances of this typo:

 ``` 
$ gg "http://bugzilla.xamarin.com" | grep https | wc -l
42
```
I originally copied the text from mtouch's doc.

Should I fix them all here or another PR?

Reply:
I'm just trying to avoid adding it in new code. Eventually my plan is to fix this everywhere (we're using http in a lot of code as well).

Reply:
Fair enough. Fixing.